tests/ConvertToSARIF.Engine.Test/SARIFConverterTests.cs

```diff
@@ -448,7 +448,8 @@ public void HydrateResults_Added()
                             }
                         }
                     }
-                }
+                },
+                Suppressions = new List<Suppression>()
```

Comment:
Add a small sample suppression to the test to validate that it adds it correctly. Generally whatever property exists in the diagnostic record it should be passed directly to the result as is